apiary.apib

```diff
@@ -372,12 +372,14 @@ We consider the following changes to be backwards-incompatible:
 We list all backwards-compatible additions here. These are currently available in all published versions.
 (There is also a [list of backwards-incompatible upgrades](#changelog) available, but those only apply if you [upgrade your API version](#upgrading-your-api-version).)
 
-#### May 2020
+#### June 2020
```

Comment:
Instead of tying this directly to the June release, we usually add a `Latest` header.

Reply:
Done! :heavy_check_mark: